Backend/Remora.Discord.Gateway/DiscordGatewayClient.cs

```diff
@@ -706,17 +515,12 @@ private async Task<Result> CreateNewSessionAsync(CancellationToken ct = default)
 
             if (receiveReady.Entity is IPayload<IInvalidSession> invalidSession)
             {
-                return new GatewayError
-                (
-                    "The newly created session was invalidated by Discord.",
-                    invalidSession.Data.IsResumable,
-                    false
-                );
+                return new GatewayError("Session invalidated", false, false);
```

Comment:
Why does this no longer attempt to resume?

Reply:
Using the `IsResumable` property appeared to result in a developer error... fixed 😄